src/sgx/faasm_sgx_enclave.edl

```diff
@@ -21,47 +21,151 @@ enclave{
 #endif
```

Comment:
This file was horrible to read before, and doesn't get formatted, so manually formatted it myself. I also add comments in the relevant parts that have changed in this PR.